__tests__/lib/generator/sitemapXml.test.js

```
@@ -0,0 +1,204 @@
+
+describe("generateSitemapXML", () => {
+  beforeAll(async () => {
+    global.TextEncoder = require("util").TextEncoder;
+    global.TextDecoder = require("util").TextDecoder;
+  });
+  beforeEach(() => {
+    global.fetch = jest.fn();
+    jest.clearAllMocks();
+  });
+  afterAll(() => {
+    delete global.TextEncoder;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)